src/styles/info-label.scss

```diff
@@ -3,31 +3,34 @@
 
 $block: #{$fd-namespace}-info-label;
 
-$fd-info-label-height: 1.125rem !default;
+$fd-info-label-height: 1rem !default;
 $fd-info-label-display-height: 1rem !default;
-$fd-info-label-icon-text-spacing: 0.1875rem !default;
-$fd-info-label-border-width: 0.063rem !default;
+$fd-info-label-icon-text-spacing: 0.125rem !default;
+$fd-info-label-border-width: 0.0625rem !default;
 $fd-info-label-border-style: solid !default;
 $fd-info-label-font-size: var(--sapFontSmallSize) !default;
 
 $color-accents: (
-  "1": ("background": var(--sapLegendBackgroundColor1), "border": var(--sapLegendColor1)),
-  "2": ("background": var(--sapLegendBackgroundColor2), "border": var(--sapLegendColor2)),
-  "3": ("background": var(--sapLegendBackgroundColor3), "border": var(--sapLegendColor3)),
-  "4": ("background": var(--sapLegendBackgroundColor5), "border": var(--sapLegendColor4)),
-  "5": ("background": var(--sapLegendBackgroundColor20), "border": var(--sapLegendColor5)),
-  "6": ("background": var(--sapLegendBackgroundColor6), "border": var(--sapLegendColor6)),
-  "7": ("background": var(--sapLegendBackgroundColor7), "border": var(--sapLegendColor7)),
-  "8": ("background": var(--sapLegendBackgroundColor8), "border": var(--sapLegendColor8)),
-  "9": ("background": var(--sapLegendBackgroundColor10), "border": var(--sapLegendColor9)),
-  "10": ("background": var(--sapLegendBackgroundColor9), "border": var(--sapLegendColor10))
+  "1": ("background": var(--fdInfo_Label_Background_Color1), "border": var(--fdInfo_Label_Border_Color1)),
+  "2": ("background": var(--fdInfo_Label_Background_Color2), "border": var(--fdInfo_Label_Border_Color2)),
+  "3": ("background": var(--fdInfo_Label_Background_Color3), "border": var(--fdInfo_Label_Border_Color3)),
+  "4": ("background": var(--fdInfo_Label_Background_Color4), "border": var(--fdInfo_Label_Border_Color4)),
+  "5": ("background": var(--fdInfo_Label_Background_Color5), "border": var(--fdInfo_Label_Border_Color5)),
+  "6": ("background": var(--fdInfo_Label_Background_Color6), "border": var(--fdInfo_Label_Border_Color6)),
+  "7": ("background": var(--fdInfo_Label_Background_Color7), "border": var(--fdInfo_Label_Border_Color7)),
+  "8": ("background": var(--fdInfo_Label_Background_Color8), "border": var(--fdInfo_Label_Border_Color8)),
+  "9": ("background": var(--fdInfo_Label_Background_Color9), "border": var(--fdInfo_Label_Border_Color9)),
+  "10": ("background": var(--fdInfo_Label_Background_Color10), "border": var(--fdInfo_Label_Border_Color10))
 );
 
 .#{$block} {
   @include fd-reset();
   @include fd-ellipsis();
 
-  border-radius: 1.125rem;
+  display: inline-flex;
+  align-items: center;
+  border-radius: 0.5rem;
+  line-height: normal;
   height: $fd-info-label-height;
   min-width: $fd-info-label-height;
```

Comment:
I haven't seen in the specs value for `min-width`

Reply:
removed